apps/mobile-wallet/src/contexts/SendContext.tsx

```diff
@@ -72,7 +72,8 @@ const initialValues: SendContextValue = {
 const SendContext = createContext(initialValues)
 
 export const SendContextProvider = ({ children }: { children: ReactNode }) => {
-  const requiresAuth = useAppSelector((s) => s.settings.requireAuth)
+  const biometricsRequiredForTransactions = useAppSelector((s) => s.settings.requireAuth)
```

Comment:
I have not yet changed the name of the following settings yet because it will create unnecessary additional code and migrations. I would rather keep the scope as minimal as possible for now.

```
requireAuth    => biometricsRequiredForTransactions
usesBiometrics => biometricsRequiredForAppAccess
```

Reply:
So when do you suggest we proceed to the rename? Following release?

Reply:
Separate PR and yes, also following release.